Octokit/Models/Response/PullRequestReview.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Diagnostics;
+using System.Globalization;
+using Octokit.Internal;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class PullRequestReview
+    {
+        public PullRequestReview() { }
+
+        public PullRequestReview(int id)
+        {
+            Id = id;
+        }
+
+        public PullRequestReview(int id, string commitId,  User user, string body, string htmlUrl, string pullRequestUrl, string state)
+        {
+            Id = id;
+            CommitId = commitId;
+            User = user;
+            Body = body;
+            HtmlUrl = htmlUrl;
+            PullRequestUrl = pullRequestUrl;
+            State = state;
+        }
+        
+        /// <summary>
+        /// The comment Id.
+        /// </summary>
+        public int Id { get; protected set; }
```

Comment:
Im thinking this Id field should be `long` rather than `int` so that we don't have growing pains in the future